_data/chains/eip155-42161.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Arbitrum One",
+  "chainId": 42161,
+  "shortName": "arb1",
+  "chain": "ETH",
+  "network": "mainnet",
+  "networkId": 42161,
+  "nativeCurrency": {
+    "name": "Ether",
+    "symbol": "ETH",
```

Comment:
The symbol cannot be ETH. We asked the community and only Ethereum L1 should be able to have the ETH symbol. Maybe use AETH?

Reply:
Thank you

Reply:
I see, I'm curious as to what the rationale behind this is.
Is there an ethereum magicians thread or something of the kind I could read up on?

Reply:
I ran this poll on twitter as it is more about users than about devs: https://twitter.com/mr_ligi/status/1393135790948036610
there are some reasons in the replies.

Reply:
There is no rational behind any of this.... That's the beauty of it

Reply:
*?*

Reply:
@ligi I'm not sure Arbitrum necessarily fits under the same criteria here. For a typical bridge, it definitely makes sense to restrict it to wrapped representations when denominating in ETH. However, given that Arbitrum is an L2 roll-up solution it's fair to say this is the same ETH as what you have on Ethereum. The reason being that you can never lose the ETH in your Arbitrum wallet. Even if the sequencer goes down, transactions can be posted to the L1 contract by other parties. This allows withdrawals to always be possible.

Reply:
What is if there is a bug in the Arbitrum (or any L2) contracts? I still think there is a difference between ETH and L2-ETH - but I see your point that L2-ETH is closer to ETH than other derivatives. Ideas welcome on how to convey this to users - but IMHO we cannot call it ETH.

Reply:
Chain-list uses ARETH for arbitrum

Reply:
What do you refer to as Chain-list?